app/src/components/account/UserMenu.vue

```diff
@@ -164,8 +164,7 @@ export default {
         networkId: address.networkId,
       })
     },
-    signOut() {
-      this.$store.dispatch(`signOut`, this.network)
+    signOut(address) {
```

Comment:
you're passing in an address but not using it?

Reply:
ping?

Reply:
it is not there anymore. Outdated. I am just not used to the "Resolve" thing. Discovering it right now